housekeeping-vacuum-tool/src/main/scripts/common.sh

```diff
@@ -33,6 +33,12 @@ else
   export HOUSEKEEPING_CLASSPATH=$HOUSEKEEPING_CLASSPATH:$HOUSEKEEPING_LIBS
 fi
 
+HOUSEKEEPING_CONF=$HOUSEKEEPING_TOOL_HOME/conf/
+
+if [[ ! -z ${HOUSEKEEPING_CONF-} ]]; then
```

Comment:
this checks if  `HOUSEKEEPNG_CONF` has a value right but won't it always because we hardcode `/conf/` to it?

Reply:
Actually, yeah, you're right; I can export `HOUSEKEEPING_CLASSPATH` outside the `if` statement.